plugins/bostonbeertime.py

```diff
@@ -0,0 +1,107 @@
+import datetime
+import pytz
+from random import randint
+
+from will.plugin import WillPlugin
+from will.decorators import respond_to
+
+class BostonBeerTimePlugin(WillPlugin):
+    @respond_to("bostonbeer")
```

Comment:
`@respond_to("(^|[^a-zA-Z])boston\s*beer(time)?($|[^a-zA-Z])")` would make it rape it little fewer ants